plugins/chat/assets/javascripts/discourse/admin-chat-route-map.js

```diff
@@ -7,7 +7,7 @@ export default {
       { path: "hooks" },
       function () {
         this.route("new");
-        this.route("show", { path: "/:id" });
+        this.route("edit", { path: "/:id" });
```

Comment:
I made this comment on the AI PR too, but it would be good to standardize on `/:id/edit` for the paths